cpp/ql/src/semmle/code/cpp/Declaration.qll

```diff
@@ -211,6 +211,26 @@ class Declaration extends Locatable, @declaration {
   /** Holds if this declaration is a member of a class/struct/union. */
   predicate isMember() { hasDeclaringType() }
 
+  /** Holds if this declaration is a member and has an lvalue ref-qualifier */
+  predicate isLValueQualified() {
+    isMember() and hasSpecifier("lvalue_ref_qualifier")
```

Comment:
The other `specifiers` names we have correspond somewhat to the source language constructs, I think. Could we replace `lvalue_ref_qualifier` with `&`? And `rvalue_ref_qualifier` with `&&`?

Reply:
That is certainly better! Thanks a lot for the suggestion :)